backend/data.go

```diff
@@ -109,8 +114,18 @@ type DataResponse struct {
 	// The data returned from the Query. Each Frame repeats the RefID.
 	Frames data.Frames
 
-	// Error is a property to be set if the the corresponding DataQuery has an error.
+	// Error is a property to be set if the corresponding DataQuery has an error.
 	Error error
+
+	// Status codes map to HTTP status values
+	Status Status
```

Comment:
I don't have any real issue with this - I suppose fine, just seems weird we want to manage our own list of the acceptable HTTP status codes  ¯\_(ツ)_/¯

Reply:
I guess the idea is to document error statuses that we want to encourage usage of and/or provide some a guidance in the SDK. This still allow flexibility in case people really wanted to use custom status codes

Reply:
Okay so sounds like we've reached a consensus? :tada: